src/io/ioexternalbus.ui

```diff
@@ -65,6 +72,9 @@
          <property name="readOnly">
           <bool>true</bool>
          </property>
+         <property name="text">
+          <string>-</string>
+         </property>
         </widget>
```

Comment:
You could add a new read-only `QTextEdit` here for displaying messages emitted by the `XTcpSocket`.

Reply:
I added a spin widget to select the port number, but I could not align it, I'm not familiar with the QTCreator.